frontend/src/components/robotHardware/index.js

```diff
@@ -0,0 +1,150 @@
+
+import { Divider, Grid, SvgIcon, Typography } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import { observer } from 'mobx-react'
+import Widget, { WidgetTitle } from '../widget'
+import { ReactComponent as LightningIcon } from './lightning.svg'
+import {
+  CircularProgressbarWithChildren,
+  buildStyles
+} from 'react-circular-progressbar'
+import 'react-circular-progressbar/dist/styles.css'
+
+const BatteryLevelDivider = observer(({left = '0'}) => {
+  const theme = useTheme()
+
+  return <Divider
+    orientation='vertical'
+    variant='middle'
+    sx={{
+      height: '12px',
+      backgroundColor: theme.palette.common.white,
+      borderColor: theme.palette.hr.batteryLevel,
+      margin: 0,
+      position: 'absolute',
+      top: '3px',
+      left,
+      marginLeft: '-1px',
+    }}
+  />
+})
+
+const Battery = observer(() => {
+  const theme = useTheme()
+  // TODO: define logic for percentage in a store (data, requests to DB, etc.)
+  const percentage = 80
+  
+  let color = theme.palette.success.main
+  let amount = 'High'
+
+  if (percentage < 20) {
+    color = theme.palette.error.main
+    amount = 'Low'
```

Comment:
Do we use these constants anywhere else? If yes, it'd make sense to make an enum.

Reply:
no, we don't use them anywhere else